src/main/resources/liquibase/202405151547-update-events-azure-billable-account-id.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.20.xsd">
+  <changeSet id="202405151547-01" author="wpoteat">
+    <comment>
+      Copy data from azure_subscription_id field to billable_account_id field
+    </comment>
+    <sql dbms="postgresql">
+      update events
+      set data = jsonb_set(data::jsonb, '{"billable_account_id"}', data -> 'azure_subscription_id')
```

Comment:
@wottop I think we need to do the same for subscriptions,contracts,tally_snapshots, billable_usage_remittances and hosts/host_tally_buckets. I would also make sure to use billing_provider='azure' in the where clause  so we limit the query because those can be big tables.